src/opm/output/eclipse/AggregateMSWData.cpp

```diff
@@ -489,6 +501,61 @@ namespace {
             };
         }
 
+        float valveFlowUnitCoefficient(const Opm::UnitSystem::UnitType uType)
+        {
+            using UType = Opm::UnitSystem::UnitType;
+
+            switch (uType) {
+                case UType::UNIT_TYPE_METRIC:
+                    return 1.340e-15f;
```

Comment:
Could we have a comment de-mystifying these constants?

Reply:
I'm inclined to say "no".  The values here are lifted straight from written sources and there's no explanation of their origins.  We only really know that the values depend on which unit convention is active.